transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_rc_performance_events.sql

```diff
@@ -0,0 +1,13 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['received_at_date'],
+    })
+}}
+
+SELECT * FROM 
+{{ ref('stg_mm_telemetry_rc__performance_events') }} 
+
+{% if is_incremental() %}
+    WHERE received_at_date >= (SELECT MAX(received_at_date) FROM {{ this }}) 
```

Comment:
Same here. `received_at` seems like a better candidate.

Reply:
what would the cluster key be in this case?

Reply:
Nice question. The clustering key may remain as is for int models. Since `received_at_date` is calculated from `received_at`, the query optimizer might take advantage of it. However we'll have to try it to see it.

It is also possible to take advantage of it to prune the data to be added using the `received_at_date`, and then implement the `append` strategy using `received_at`. But this is more challenging to implement and IMHO a premature optimization. Let's monitor incremental logic performance and implement it on a second stage if needed.

Reply:
Updated the strategy to append using `received_at`